### PR TITLE
Add ruby:master-debug-nightly-bionic

### PR DESCRIPTION
**Pull request by ko1**
I'm Koichi working on MRI development.

We prepared `ruby:master-debug-nightly-bionic` docker image which provides MRI compiled with enabling assertions.
Assertions check the healthiness of an interpreter internal state more aggressively, but it is slow than normal built interpreter.

Of course we are using this assertion enabled interpreter for MRI's CI.
However we want to try further more tests on assertion enabled interpreter.
If Rails framework, the big and most popular Ruby software try it, it will be huge help for MRI development.
The drawback is more CPU time is needed.

Thank you for your consideration,
Koichi

### CODE REVIEW

**Comment by ko1** [comment]
related link: https://bugs.ruby-lang.org/issues/16895

File: pipeline-generate [hunk 1]
```diff
@@ -73,8 +73,8 @@ end
 
 ONE_RUBY = RUBIES.last || SOFT_FAIL.last
 
-MASTER_RUBY = "rubylang/ruby:master-nightly-bionic"
-SOFT_FAIL << MASTER_RUBY
+MASTER_RUBIES = ["rubylang/ruby:master-nightly-bionic", "ruby:master-debug-nightly-bionic"]
```
Comment (yahonda) [inline pipeline-generate:76]: "ruby:master-debug-nightly-bionic" should be "rubylang/ruby:master-debug-nightly-bionic"

Comment (ko1) [reply pipeline-generate:76]: OMG

**Comment by matthewd** [comment]
Hey @ko1!

Yeah, we can definitely run this!

Instead of run it on each of our commits (which seems unnecessary, and would slow down our time-to-green on each PR), I suggest we run a separately scheduled ruby-debug build. Do you know what time it would be best to schedule that, to have the best chance of always getting the fresh nightly? From https://hub.docker.com/r/rubylang/ruby/tags, maybe 1800 UTC / 0300 JST is a good choice?

**Comment by ko1** [comment]
> maybe 1800 UTC / 0300 JST is a good choice?

Absolutely.

**Comment by zzak** [comment]
I think this would still be useful, but I'm not sure how to achieve what @matthewd is suggesting with buildkite. It seems like there is only a UI to activate scheduled builds?

**Comment by rafaelfranca** [comment]
Hey @ko1, I added the debug build with #84.

You can see it on https://buildkite.com/rails/rails-nightly

**Pull request closed without merging**